### DISCUSSION TITLE
module 'fluidsynth' has no attribute 'synth'

### DISCUSSION THREAD

**Discussion by Fiddleford42** [FluidSynth/fluidsynth#1428] [Q&A] [answered]
I'm assuming I'm doing something simple wrong, but I haven't been able to get fluidsynth to work in python. It always returns the error "module 'fluidsynth' has no attribute 'synth'". I am running my code in a vscode virtual environment and have done a pip install for both fluidsynth and pyfluidsynth. I haven't added a path to the sound font because I haven't managed to get past line four without an error. Does anyone know how to fix this?

This is my code:

`import time
import fluidsynth

fs = fluidsynth.Synth()
fs.start(driver = 'dsound')  # use DirectSound driver

sfid = fs.sfload(r'c:\users\me\mydir\FluidR3_GM.sf2')  # replace path as needed
fs.program_select(0, sfid, 0, 0)

fs.noteon(0, 60, 30)
fs.noteon(0, 67, 30)
fs.noteon(0, 76, 30)

time.sleep(3.0)

fs.noteoff(0, 60)
fs.noteoff(0, 67)
fs.noteoff(0, 76)

time.sleep(1.0)

fs.delete()`

**Comment by spessasus** [comment, marked as answer]
Hi,

Your question is probably related to [pyfluidsynth](https://github.com/nwhitehead/pyfluidsynth). This repo is only for the fluidsynth itself and not pyfluidsynth, so I don't think you're gonna get an answer here. I suggest you ask there instead.

**Reply by Fiddleford42** [reply]
Thanks
